### PR TITLE
CEDS-6105 Section 1 changes - Details Pattern Audit

### PR DESCRIPTION
**Pull request by Jack-Brannon**
Covers the content changes required for section 1 of 6 in the export dec.

### CODE REVIEW

File: app/views/journey/standard_or_other_journey.scala.html [hunk 1]
```diff
@@ -61,13 +51,21 @@
...
+        )))
+
         @govukRadios(Radios(
             name = "type",
             fieldset = Some(Fieldset(
                 legend = Some(Legend(
                     content = Text(messages("declaration.type.description")),
                     isPageHeading = true,
-                    classes = gdsPageLegend
+                    classes = "govuk-visually-hidden"
```
Comment (udinobi) [inline app/views/journey/standard_or_other_journey.scala.html:68]: Nit: could the `@import` for `gdsPageLegend` at L24 be removed?

File: app/views/journey/standard_or_other_journey.scala.html [hunk 2]
```diff
@@ -97,12 +95,6 @@
             content = Text(messages("declaration.type.warning"))
         ))
 
-        @govukDetails(Details(
```
Comment (udinobi) [inline app/views/journey/standard_or_other_journey.scala.html:100]: The injection of `GovukDetails` at L32 should be removed

File: app/views/journey/standard_or_other_journey.scala.html [hunk 3]
```diff
@@ -37,22 +37,12 @@
     externalLink: externalLink,
     saveAndContinue: saveAndContinue,
     formHelper: FormWithCSRF,
-    appConfig: AppConfig
```
Comment (udinobi) [inline app/views/journey/standard_or_other_journey.scala.html:40]: The injection of `GovukDetails` at L32 should be removed

File: app/views/journey/other_journey.scala.html [hunk 4]
```diff
@@ -113,12 +104,6 @@
             paragraphBody(messages("declaration.type.insetText.p2"))
         ))))
 
-        @govukDetails(Details(
```
Comment (udinobi) [inline app/views/journey/other_journey.scala.html:116]: The injection of `GovukDetails` at L30 should be removed

File: app/views/section1/confirm_ducr.scala.html [hunk 5]
```diff
@@ -90,9 +89,7 @@
             attributes = Map("id" -> "expander-ducr"),
             summary = Text(messages("declaration.confirmDucr.expander.title")),
             content = HtmlContent(expanderContent)
-        ))
-
-        @tariffExpander(ConfirmDucr, request.declarationType)(messages, appConfig)
```
Comment (udinobi) [inline app/views/section1/confirm_ducr.scala.html:95]: The injection of `AppConfig` at L42, as well as the `@import`, should be removed

File: app/views/section1/ducr_choice.scala.html [hunk 6]
```diff
@@ -72,8 +71,6 @@
             ))
         )
 
-        @tariffExpander(DucrChoice, request.declarationType)(messages, appConfig)
```
Comment (udinobi) [inline app/views/section1/ducr_choice.scala.html:75]: The injection of `AppConfig` at L39, as well as the @import, should be removed

File: app/views/section1/ducr_entry.scala.html [hunk 7]
```diff
@@ -83,8 +82,6 @@
             content = HtmlContent(expanderContent)
         ))
 
-        @tariffExpander(Ducr, request.declarationType)(messages, appConfig)
```
Comment (udinobi) [inline app/views/section1/ducr_entry.scala.html:86]: The injection of `AppConfig` at L41, as well as the `@import`, should be removed

File: app/views/section1/link_ducr_to_mucr.scala.html [hunk 8]
```diff
@@ -75,7 +75,7 @@
             ))
         )
 
-        @tariffExpander(LinkDucrToMucr, request.declarationType, Some("tariff.declaration.linkDucrToMucr.title"))(messages, appConfig)
```
Comment (udinobi) [inline app/views/section1/link_ducr_to_mucr.scala.html:78]: The injection of `tariffExpander` and `AppConfig`, as well as the `@import`, should be removed

File: app/views/section1/trader_reference.scala.html [hunk 9]
```diff
@@ -82,8 +81,6 @@
             content = HtmlContent(expanderContent)
         ))
 
-        @tariffExpander(TraderReference, request.declarationType)(messages, appConfig)
```
Comment (udinobi) [inline app/views/section1/trader_reference.scala.html:85]: The injection of `AppConfig` at L38, as well as the `@import`, should be removed

File: app/views/journey/other_journey.scala.html [hunk 10]
```diff
@@ -47,15 +47,6 @@
     )
 }
 
-@detailsText = @{
-  val detailsLink = externalLink(
-    text = messages("tariff.declaration.others.linkText.0"),
-    url = appConfig.additionalDeclarationType
```
Comment (udinobi) [inline app/views/journey/other_journey.scala.html:53]: I reckon you can also remove L53 in `conf/externalLinks.conf`. But please check before if the config property is still used somewhere else

File: conf/messages [hunk 11]
```diff
@@ -209,7 +208,6 @@ tariff.declaration.traderReference.clearance.linkText.0 = Data Element 2/1 - Sim
 tariff.declaration.ducr.common.linkText.0 = Data Element 2/1 - Simplified Declaration, Previous Documents (opens in new tab)
 tariff.declaration.lrn.common.linkText.0 = Data Element 2/5 - LRN - No previous reference (opens in new tab)
 tariff.declaration.linkDucrToMucr.title = More details about MUCR consolidation
```
Comment (udinobi) [inline conf/messages:210]: Shouldn't this message be removed?

File: conf/messages [hunk 12]
```diff
@@ -200,7 +200,6 @@ tariff.declaration.type.text.1 = A standard declaration (arrived or pre-lodged)
 tariff.declaration.type.text.2 = Find details about {0}.
```
Comment (udinobi) [inline conf/messages:200]: I think the `tariff.declaration.type.title` key at L198 can be removed.

File: app/forms/section1/DucrChoice.scala [hunk 13]
```diff
@@ -17,14 +17,5 @@
...
 
-object DucrChoice extends DeclarationPage {
-
-  override def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] =
-    decType match {
-      case CLEARANCE => List(TariffContentKey("tariff.declaration.ducr.clearance"))
-      case _         => List(TariffContentKey("tariff.declaration.ducr.common"))
-    }
-}
+object DucrChoice extends DeclarationPage {}
```
Comment (udinobi) [inline app/forms/section1/DucrChoice.scala:21]: Nit: you do not need the final braces

File: conf/messages [hunk 14]
```diff
@@ -199,18 +199,8 @@ tariff.declaration.type.title = More details about standard declarations
 tariff.declaration.type.text.1 = A standard declaration (arrived or pre-lodged) is a full declaration. This means all sections of the declaration must be completed.
 tariff.declaration.type.text.2 = Find details about {0}.
 tariff.declaration.type.linkText.2 = making a full export declaration (opens in a new tab)
```
Comment (udinobi) [inline conf/messages:201]: Are you sure you still need lines 198-201 and 203?

File: conf/messages.cy [hunk 15]
```diff
@@ -202,16 +202,9 @@ tariff.declaration.type.text.2 = Dewch o hyd i fanylion am {0}.
 tariff.declaration.type.linkText.2 = wneud datganiad allforio llawn (yn agor tab newydd)
```
Comment (udinobi) [inline conf/messages.cy:202]: Are you sure you still need lines 199-203 and 206-207?

File: app/forms/section1/DucrChoice.scala (hunk 13 above)
Comment (Jack-Brannon) [reply app/forms/section1/DucrChoice.scala]: Removed braces from here and similar files changed

File: conf/messages (hunk 14 above)
Comment (Jack-Brannon) [reply conf/messages:200]: Removed unused messages, few still in use.

File: conf/messages.cy (hunk 15 above)
Comment (Jack-Brannon) [reply conf/messages.cy:201]: Removed unused messages, few still in use.

**Review by udinobi — APPROVED** [review]

**Review by udinobi — DISMISSED** [review]

**Pull request merged**